### PR TITLE
Traumatic Shock Effects Derandomized

### PR DESCRIPTION
**Pull request by Surrealaser**
## About The Pull Request

RNG hard stuns from traumatic shock have been replaced with slowdown and stagger stacks.

## Why It's Good For The Game

Gets rid of bullshit RNG and ridiculously long stuns per our ongoing trend towards good gameplay.

## Changelog
:cl:
code: RNG hard stuns from traumatic shock have been replaced with slowdown and stagger stacks.
/:cl:

### CODE REVIEW

**Comment by TiviPlus** [comment]
@Surrealaser
A) tweak doesnt exist anymore
B) Dont use language in titles and preferably pr bodies that might get us GitHub smacked

**Comment by Surrealaser** [comment]
> @Surrealaser
> A) tweak doesnt exist anymore
> B) Dont use language in titles and preferably pr bodies that might get us GitHub smacked

Okay, changed the tag.

File: code/modules/mob/living/carbon/human/life/handle_shock.dm [hunk 1]
```diff
@@ -40,42 +40,38 @@
...
-				emote("me", 1, " is having trouble standing.")
 			blur_eyes(2)
 			stuttering = max(stuttering, 5)
-			if(prob(2))
-				if(!lying_angle)
-					emote("me", 1, " is having trouble standing.")
+			adjust_stagger(3, FALSE, 3)
+			add_slowdown(3)
+			if(prob(20))
 				to_chat(src, "<span class='danger'>[pick("The pain is excruciating", "Please, just end the pain", "Your whole body is going numb")]!</span>")
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/human/life/handle_shock.dm:51]: Why remove the emotes?

File: code/modules/mob/living/living_defines.dm [hunk 2]
```diff
@@ -127,3 +127,6 @@
 
 	/// This is the cooldown on suffering additional effects for when we exhaust all stamina
 	COOLDOWN_DECLARE(last_stamina_exhaustion)
+
+	/// This is the cooldown on suffering additional effects for when shock gets high
+	COOLDOWN_DECLARE(last_shock_effect)
```
Comment (TiviPlus) [inline code/modules/mob/living/living_defines.dm:132]: Why is this on living if seemingly only used by human?

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/life/handle_shock.dm (hunk 1 above)
Comment (Surrealaser) [reply code/modules/mob/living/carbon/human/life/handle_shock.dm:51]: May as well get rid of the spam, especially when the stun elements have been removed at this level.

Comment (TiviPlus) [reply code/modules/mob/living/carbon/human/life/handle_shock.dm:51]: This isnt problematic rng and I see no reason for removing them, add them back

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**